### PR TITLE
Replace all docs.woocommerce.com links with woocommerce.com's

### LINKED ISSUE #8096: WooCommerce Docs link on the help menu does not work
### Describe the bug

When clicking the WooCommerce Docs link on the help menu of WooCommerce Admin, it redirects the page to https://woocommerce.com/?utm_source=help_panel&utm_medium=product instead of https://woocommerce.com/documentation/.

### To Reproduce

1. Go to /wp-admin/admin.php
2. Click the WooCommerce Docs  link on the Help menu

### Actual behavior:

 It redirects the page to ‘https://woocommerce.com/’ 

### Screenshots

Screen Shot 2021-12-29 at 16 10 11

### Expected behavior

redirect to https://woocommerce.com/documentation

### Additional context

Details: p1640760538072300-slack-C4E0DVDB2 | pb22l9-14Q-p2#comment-1085

### PR DESCRIPTION
**Pull request by chihsuan**
Fixes #8096

This PR fixes the broken "WooCommerce Docs" link and replaces all old docs.woocommerce.com links with woocommerce.com's.

### Detailed test instructions:

1. Go to /wp-admin/admin.php
2. Click "Help menu" on the top-right corner and make sure all links work
3. Confirm the other links still work (You could use an editor like vscode to help you open the links without copy-and-paste)

### CODE REVIEW

File: src/Marketing/InstalledExtensions.php [hunk 1]
```diff
@@ -275,7 +275,7 @@ protected static function get_amazon_ebay_extension_data() {
 			}
 
 			$data['settingsUrl'] = admin_url( 'admin.php?page=codisto-settings' );
-			$data['docsUrl']     = 'https://docs.woocommerce.com/document/getting-started-with-woocommerce-amazon-ebay-integration/?utm_medium=product';
+			$data['docsUrl']     = 'https://woocommerce.com/documentation/getting-started-with-woocommerce-amazon-ebay-integration/?utm_medium=product';
```
Comment (chihsuan) [inline src/Marketing/InstalledExtensions.php:278]: Though it's not in the allowed plugin list, this plugin seems no longer available and the document link is 404.

I think this is the new plugin by Codisto.
https://woocommerce.com/products/amazon-ebay-integration/

Comment (chihsuan) [reply src/Marketing/InstalledExtensions.php:278]: If we're longer need this. I think we can just remove it. Please help confirm this, thanks.

Comment (moon0326) [reply src/Marketing/InstalledExtensions.php:278]: @chihsuan I think you're correct on it. I installed `https://woocommerce.com/products/amazon-ebay-integration/` and confirm the plugin has the same setting page path (page=codisto-settings).

**Review by moon0326 — APPROVED** [review]
Thank you for working on this so quickly @chihsuan 

It LGTM and tested well 🚢 🚀

File: src/Marketing/InstalledExtensions.php (hunk 1 above)
Comment (ilyasfoo) [reply src/Marketing/InstalledExtensions.php:278]: @chihsuan Actually I've found the new documentation page in https://woocommerce.com/document/multichannel-for-woocommerce-google-amazon-ebay-walmart-integration/ from the product page. I think we can replace it with this instead

Comment (chihsuan) [reply src/Marketing/InstalledExtensions.php]: Updated. Thanks for confirming! 😄

**Pull request merged**